### PR TITLE
Step two

### PR DESCRIPTION
**Pull request by jCabreraAltimetrik**
Would you mind check this out ?

### CODE REVIEW

File: force-app/main/default/classes/TaskTriggerHandler.cls [hunk 1]
```diff
@@ -1,53 +1,47 @@
...
     public static void checkOnCompleteTask(List<Task> taskList){
-        List<Contact> toUpdate = [Select Id, Name, Points__c,
-                                  	(
-                                    	Select Id,ActivityDate,CreatedDate ,Task_Type__c,Points__c,IsClosed
-                                        FROM Tasks
-                                        WHERE Id IN :taskList
-                                        AND IsClosed = true
-                                    )	
-                                  FROM Contact];
+        List<Contact> selectedContacts = [Select Id, Name, Points__c,
```
Comment (stefaniaoverman) [inline force-app/main/default/classes/TaskTriggerHandler.cls]: this is a better name for the variable =)

File: force-app/main/default/classes/TaskTriggerHandler.cls [hunk 2]
```diff
@@ -1,53 +1,47 @@
...
+                                        (
+                                            Select Id,ActivityDate,CreatedDate ,Task_Type__c,Points__c,IsClosed
+                                            FROM Tasks
+                                            WHERE Id IN :taskList
+                                            AND IsClosed = true
+                                        )	
+                                      	FROM Contact];
         
-        for(Contact c : toUpdate)
+		List<Contact> toUpdate = new List<Contact>();
```
Comment (stefaniaoverman) [inline force-app/main/default/classes/TaskTriggerHandler.cls:13]: you could improve the name of this variable (in future commits). something like: contactsToUpdate

File: force-app/main/default/classes/TaskTriggerHandler.cls [hunk 3]
```diff
@@ -1,53 +1,47 @@
...
+                    contactPoints += processTaskPoints(task);
                 }
                 c.Points__c = contactPoints;
+                toUpdate.add(c);
             }
         }      
-        update toUpdate;
+        if(toUpdate.size() >= 1)
+        {
+             update toUpdate;
```
Comment (stefaniaoverman) [inline force-app/main/default/classes/TaskTriggerHandler.cls:31]: if you improve the name of the variable, then you wont have "update toUpdate" hehe

**Review by stefaniaoverman — COMMENTED** [review]
just two extra comments (on a single topic) that you can improve in future commits =). Well done

**Comment by jCabreraAltimetrik** [comment]
Corrected code as required.

**Pull request closed without merging**